modules/xfem/include/userobjects/LevelSetCutUserObject.h

```diff
@@ -38,6 +38,8 @@ class LevelSetCutUserObject : public GeometricCutUserObject
   virtual const std::vector<Point>
   getCrackFrontPoints(unsigned int num_crack_front_points) const override;
 
+  virtual bool getConsistentSideBoolean(const Node * node) const override;
```

Comment:
Add doxygen

Reply:
This is overriding the function in the parent class. Do I still need to add doxygen here?

Reply:
Well I added a doxygen anyways.